doc/interfaces/nixnet.rst

```diff
@@ -0,0 +1,22 @@
+NI-XNET
```

Comment:
Oh, and you also need to add this to interfaces.rst.

Reply:
Done!